content/docs/07-reference/ai-sdk-rsc/11-use-streamable-value.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: useStreamableValue
+description: Reference for the useStreamableValue function from the AI SDK RSC
+---
+
+# `useStreamableValue`
+
+It is a React hook that takes a streamable value created using [`createStreamableValue`](/docs/reference/ai-sdk-rsc/create-streamable-value) and returns the current value, error, and pending state.
+
+## Import
+
+<Snippet text={`import { useStreamableValue } from "ai/rsc"`} prompt={false} />
+
+## Example
+
+This is useful for consuming streamable values received from a component's props.
+
+```tsx
+function MyComponent({ streamableValue }) {
+  const [data, error, pending] = useStreamableValue(streamableValue);
+
+  if (pending) return <div>Loading...</div>;
+  if (error) return <div>Error: {error.message}</div>;
+
+  return <div>Data: {data}</div>;
+}
+```
+
+<ObjectTableList
```

Comment:
I've switched to using `PropertiesTable` and explicit headings so we get a correct in-page navigation on the right.

E.g.:
```

## API Signature

### Parameters

<PropertiesTable
  content={[
 ```
 
 See https://raw.githubusercontent.com/vercel/ai/main/content/docs/07-reference/ai-sdk-core/01-generate-text.mdx for an example